app/services/cfe/submit_assets_service.rb

```diff
@@ -31,15 +32,16 @@ def call(cfe_assessment_id)
           value: property_entry_form.house_value,
           outstanding_mortgage: (property_entry_form.mortgage if property_form.owned_with_mortgage?) || 0,
           percentage_owned:,
+          subject_matter_of_dispute: (property_entry_form.house_in_dispute && !estimate.upper_tribunal?) || false,
```

Comment:
because we have `!estimate.upper_tribunal?` in so many places, I wonder if it's worth pulling it into its own method? Or own module?